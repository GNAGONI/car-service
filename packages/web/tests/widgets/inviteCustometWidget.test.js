import React from 'react'
import renderer from 'react-test-renderer'
import InviteCustometWidget from '../../components/widgets/inviteCustometWidget'

it('renders correctly', () => {
  const tree = renderer.create(<InviteCustometWidget />).toJSON()
  expect(tree).toMatchSnapshot()
})
