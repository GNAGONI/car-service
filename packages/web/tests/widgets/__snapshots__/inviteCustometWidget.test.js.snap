// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="widget invite-widget"
>
  <span
    className="price-tag"
  >
    <span
      className="price"
    >
      Just 
      <br />
      Â£9.99
    </span>
  </span>
  <div
    className="widget-header text-center"
  >
    <h3>
      Invite your customers to leave a review
    </h3>
    <p>
      Gain that extra level of trust
    </p>
  </div>
  <ul
    className="star-list"
  >
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="description"
          height="41"
          src=""
          width="41"
        />
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="description"
          height="41"
          src=""
          width="41"
        />
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="description"
          height="41"
          src=""
          width="41"
        />
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="description"
          height="41"
          src=""
          width="41"
        />
      </span>
    </li>
  </ul>
  <a
    className="widget-btn btn btn-dark d-block"
    href="#"
  >
    Invite customers
  </a>
</section>
`;
